### ISSUE TITLE
"Network Error - 99 - TLS initialization failed"

### ISSUE THREAD

**Issue by Twi-Hard** [Sibusten/derpibooru-downloader#20]
I get that error with both my old installation and a fresh installation when I try to download something.

**Comment by Sibusten** [comment]
What version of the program are you using?
64 bit or 32 bit?
Are you running the program with all files exactly as they are in the zip?
If you are running the program from a shortcut or task scheduler, are you making sure the working directory is set to the folder containing the program?

**Comment by Twi-Hard** [comment]
My main copy of it was version 1.4.2. After I got the error I upgraded it to version 1.4.3 but it still didn't work so I tried using a new copy of 1.4.3 in a new folder with the old .ini file.
I just reinstalled Windows after using Linux for a couple of days. Windows was fully up to date before I attempted downloading. The downloader is stored on an external drive so I don't know what's different.
It's the 64-bit version (DerpibooruDownloader_v1.4.3.7z).
I originally got the error when I tried to update the archive with the GUI exe. While I thought that was going I set up the task scheduler tasks but neither the GUI or the CMD version worked. I didn't use shortcuts.
Everything was working before I reinstalled, I didn't think the problems were related.
![image](https://user-images.githubusercontent.com/6430050/56644619-e2bcd400-6630-11e9-8725-a4d632d1ed40.png)
![image](https://user-images.githubusercontent.com/6430050/56644724-18fa5380-6631-11e9-85b8-6a5ccf092faf.png)
